rudder-lang/src/error.rs

```diff
@@ -205,3 +209,50 @@ impl fmt::Display for Error {
         }
     }
 }
+
+
+// Must have a trait to add the following method as ngrammatic::corpus only has an `add_text()` method
+trait FuzzyChecker<'src, I> {
+	fn add_vec(&mut self, list: I);
+}
+
+impl<'src, I: Iterator<Item = &'src Token<'src>>> FuzzyChecker<'src, I> for Corpus {
```

Comment:
It works here, but usually having a life time "&'a Object<'a>" often ends up with the constraint beging 'a == 'static 
which can become problematic, so avoid it when possible